abap-code-review/src/introduction.adoc

```diff
@@ -0,0 +1,40 @@
+[cols="1,3",frame=none,grid=none]
+|===
+|Home
+|link:https://github.com/SAP/styleguides[https://github.com/SAP/styleguides]
+
+|License
+|link:https://github.com/SAP/styleguides/blob/main/LICENSE[Creative Commons]
+
+|Build
+|{docdatetime}
+|===
+
+== Introduction
+
+Code review is a measure for assuring software quality by letting developers read the code of others. While the primary goal is to increase quality by finding defects, it also fosters learning and shared code ownership.
+
+link:https://help.sap.com/viewer/Continuous-Integration-and-Delivery-Introduction-Guide/5ba483a2c97b4ad5ab0148f4a6c5a9ee.html[Continuous integration / continuous delivery] (CI/CD) is commonly seen as a prerequisite to faster delivery of cloud-based products. This comprises the ability to review, test, and, if required, veto changes before they affect every other developer working on the same repository.  This capability to validate a change in _isolation_ before integrating it is also known as link:https://trunkbaseddevelopment.com/game-changers/#google-s-internal-devops-2006-onwards[proposed commits] or link:https://martinfowler.com/bliki/PendingHead.html[pending head]. For instance, most git platforms have implemented this concept as depicted below.
+
+[plantuml,activity,svg,align="center"]
+....
+@startuml
+skinparam monochrome true
+
+start
+:Request;
+repeat
+  :Automated Review;
+  :Peer Review;
+backward:Rework;
+repeat while (OK?) is (No)
+->Yes;
+:Submit(merge);
+stop
+@enduml
+....
+
+Once your source repository supports such a validation, manual change-based code reviews usually come for free because it does not matter technically whether a change is vetoed by a human vote or an automated test verdict.
```

Comment:
I don't think a source repository supports validation but source code management system do:
https://de.wikipedia.org/wiki/Source_Code_Control_System

Reply:
Done. VCS is another term for the same.